backend/src/main/java/com/google/models/RelatedData.java

```diff
@@ -26,7 +26,7 @@
 import java.time.format.DateTimeParseException;
 
 /** 
- * Store related-data and metadata. 
+ * Store related data with data points and meta data.
```

Comment:
I meant like, add more details about what the related data is for, like
"Store related data for anomalies with data points and metadata"

Reply:
ohh, i see. made the change!